adal/devApps/XFormsApp/SecondPage.cs

```diff
@@ -37,6 +37,13 @@ public class SecondPage : ContentPage
     {
         private readonly StringBuilder _logs = new StringBuilder();
 
+        static string ClientId = "de49ddaf-c7f8-4a06-8463-3c6ae124fe52";
+
+        //static string AndroidBrokerRedirectURI = "msauth://com.microsoft.xformsdroid.adal/mJaAVvdXtcXy369xPWv2C7mV674=";
```

Comment:
this is the redirectUri for broker for this application.

Reply:
any reason this is not either left there or simply deleted? Ideally it should be kept and a button in the ui should be able to switch between broker and non-broker. thoughts?

---
In reply to: [203584400](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1140#discussion_r203584400) [](ancestors = 203584400)

Reply:
That's a good idea to include a button. I will do that.

Reply:
done.